### ISSUE TITLE
adding variant ships to shipyards

### ISSUE THREAD

**Issue by daggs1** [endless-sky/endless-sky#7123]
### Problem Description

I'm working on some new variant ships feature which I want to add to a specific shipyard.
is there a proper way todo so?

### Related Issue Links

N/A

### Desired Solution

the ship can be purchased from the shipyard

### Alternative Approaches

I can implement the code is it is missing

**Comment by warp-core** [comment]
Yes, you can add ship variants to shipyards. Just use the variant name instead of the base ship name.
Like this:
```
shipyard "whatever"
	"Auxiliary (Cargo)"
	"Auxiliary (Transport)"
	"Auxiliary"
```

**Comment by daggs1** [comment]
interesting, I added them under new type of shipyard and I see this:
> Warning: shipyard "Lionheart Custom" is referred to, but has no ships.
and I don't see them in the shipyard

**Comment by warp-core** [comment]
Can you send the shipyard definition here?

**Comment by daggs1** [comment]
> shipyard "Lionheart Custom"
> 	"Aerie (No Bays)"
> 	"Bactrian (No Bays)"
>

**Comment by warp-core** [comment]
It looks like you're not indenting the ship name lines properly.

Also, do you use Discord?

**Comment by daggs1** [comment]
> It looks like you're not indenting the ship name lines properly.
the indent is ok, the paste is bad
> Also, do you use Discord?
no

**Comment by warp-core** [comment]
Try using \`\`\` before and after the data instead of `>` so the formatting doesn't get ruined. That's how I pasted the example here.

**Comment by daggs1** [comment]
```
shipyard "Lionheart Custom"
	"Aerie (No Bays)"
	"Bactrian (No Bays)"

**Comment by warp-core** [comment]
Can you send the ship definitions the same way?

**Comment by daggs1** [comment]
```
ship "Aerie" "Aerie (No Bays)"
	attributes
		"cost" 4900000
		"cargo space" 160
		"drag" 4.1
		"outfit space" 390
		"weapon capacity" 150
		"engine capacity" 95
	-bay
	-bay

ship "Bactrian" "Bactrian (No Bays)"
	attributes
		"cost" 24640000
		"cargo space" 695
		"drag" 16.1
		"outfit space" 740
		"weapon capacity" 300
		"engine capacity" 180
	-bay
	-bay
	-bay
